ams/server/src/main/java/com/netease/arctic/server/table/DefaultTableService.java

```diff
@@ -308,7 +309,10 @@ public void initialize() {
     });
 
     if (headHandler != null) {
-      headHandler.initialize(tableRuntimeMetaList);
+      // Paimon table don't need optimizing in AMS
+      List<TableRuntimeMeta> noPaimonMetas =
+          tableRuntimeMetaList.stream().filter(t -> t.getFormat() != TableFormat.PAIMON).collect(Collectors.toList());
+      headHandler.initialize(noPaimonMetas);
     }
```

Comment:
What about table maintain actions? 
Would it be better to place this check in DefaultOptimizingServices?

Reply:
The modification in this area is being pushed forward, so we will do a simple adaptation here for now.